### PR TITLE
More specific error tests

### PR DESCRIPTION
**Pull request by darinkrauss**
@jh-bate I did this last week, but apparently forgot to push it up and have you do a PR. It is only changes to tests.

### CODE REVIEW

File: data/normalizer/standard_test.go [hunk 1]
```diff
@@ -36,9 +36,9 @@ var _ = Describe("Standard", func() {
 			var err error
 			standardContext, err = context.NewStandard(test.NewLogger())
 			Expect(standardContext).ToNot(BeNil())
-			Expect(err).ToNot(HaveOccurred())
+			Expect(err).To(Succeed())
```
Comment (jh-bate) [inline data/normalizer/standard_test.go]: so a `Succeed()` error just means it empty?

**Comment by jh-bate** [comment]
LGTM, just one question

File: data/normalizer/standard_test.go (hunk 1 above)
Comment (darinkrauss) [inline data/normalizer/standard_test.go]: Yes, `To(Succeed())` is the same as `ToNot(HaveOccurred())` and both mean that the error is nil.

Comment (jh-bate) [inline data/normalizer/standard_test.go]: `ToNot(HaveOccurred())` reads better to me

Comment (darinkrauss) [inline data/normalizer/standard_test.go]: Okay, potatoes vs. potAtoes to me. I'll change it when I'm in there next.

**Pull request merged**